LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-root-expected.html

```diff
@@ -35,5 +34,16 @@
   color: white;
```

Comment:
nit: weird indent (2 instead of 4 space)

Reply:
Fixed.